tck/README.adoc

```diff
@@ -28,65 +28,87 @@ must be executed to validate compliance.
 The optional TCK currently validates the following non-mandatory features:
 
 - REST stat metrics
+- Base metrics
 
 == Dependencies
 
 To enable the tests in your Arquillian based test project you need to add the following dependencies:
 
 [source, xml]
 ----
+
+<properties>
+    <version.microprofile.metrics>5.0.0</version.microprofile.metrics>
```

Comment:
Don't do this as they were defined by microprofile-parent pom.xml

Reply:
This is for the README for tck runner